docs/testing-debug-phpunit.md

```diff
@@ -129,6 +129,32 @@ Congratulations, you have configured the IDE to run the tests.
 ## 5. Let's debug the tests
 
 - Tell DDev to use XDebug: `ddev xdebug on`.
+- Confirm the port xdebug is listening to `/admin/reports/status/php`, look for xdebug_client_port
+
+### VS Code
+- Install the [PHP Debug extension](https://marketplace.visualstudio.com/items?itemName=xdebug.php-debug)
+- Initialize the project for PHP Debugging by selecting “create a launch.json file” and then “PHP” from the Select Debugger options. This action will create the .vscode/launch.json file.
+- Open the launch.json and set the port and the pathMappings as follows:
+
+```
+"configurations": [
+    {
+      "name": "Listen for Xdebug",
+      "type": "php",
+      "request": "launch",
+      "port": 9003,
```

Comment:
```suggestion
      "hostname": "0.0.0.0",
      "port": 9003,

```

Reply:
Thank you @rfay I didn't even look to see if this existed already